ONSdigital/response-operations-ui
# Tbl directly use uaa sign in

TBL to get response operations to directly sign in using UAA instead of through backstage.

## How to Test
- Check if you can successfully sign into internal account (uaa_user)
- Check you get error banner if wrong password entered
- Check you can view your secure messages once logged in

## Config updates

Updated confluence: https://digitaleq.atlassian.net/wiki/spaces/RASB/pages/406257665/Pending+Release+Config+Changes
Updated docker dev: https://github.com/ONSdigital/ras-rm-docker-dev/pull/54

Updated concourse pipeline: https://github.com/ONSdigital/ras-deploy/pull/9

https://trello.com/c/oodNbXR2/100-response-operations-ui-should-sign-in-directly-using-uaa-not-via-backstage
